themes/formulize_standalone/theme.html

```diff
@@ -134,6 +134,7 @@
     <br /> *}>
     <{$icms_footer}>
     <{ if $masquerade_username }>
+        <br>

```

Comment:
This puts the Revert button on a separate line, which I don't want, especially because then it appears by default just below the bottom of the page. I don't mind that the other footer text isn't in the absolute middle. The revert button works in Chrome and Firefox, but it doesn't seem to work in Safari. You may have to clear templates and force-refresh to get new CSS rules.